src/ResourceManagement/HDInsight/HDInsight/CustomizationModels/ClusterCreateParameters.cs

```diff
@@ -194,6 +194,11 @@ public partial class ClusterCreateParameters
         public SecurityProfile SecurityProfile { get; set; }
 
         /// <summary>
+        /// Gets or sets the worker node datadisksgroupproperties for worker nodes. Optional. For Kafka clusters only
```

Comment:
This is ambiguous. Is it required for Kafka or optional in any case?
If optional for any cluster:
Optional. Gets or sets the worker node DataDisksGroupProperties for worker nodes. Used for Kafka clusters only.

If it's required for Kafka clusters:
Required for Kafka clusters. Gets or sets the worker node DataDisksGroupProperties for worker nodes.